packages/yourloops/pages/profile/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2021, Diabeloop
+ * Profile index
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ */
+
+import { withRouter } from 'react-router-dom';
+import { ProfilePage } from './profile';
+
+export default withRouter(ProfilePage);
```

Comment:
withRouter seems useless.

```suggestion
import { ProfilePage } from './profile';
export default ProfilePage;
```
Maybe doing an export default for ProfilePage  ?